blog/views.py

```diff
@@ -80,3 +80,18 @@ def register_user(request):
             return HttpResponse("Something went wrong, please try again")
 
     return render(request, "blog/register_user.html", context)
+
+
+def login_user(request):
+    if request.method == "POST":
+        username = request.POST["username"].lower()
```

Comment:
When a user logs in, post method starts and looks for the selected username in the database.

Referring to: https://github.com/jashchanin/purokomp-app/pull/6/commits/e1e493b96b0c33ec997b597c7cf7c625e9554613#r1020796973
No matter how user types the username, his username will always be lowercase with this line. That prevents some case sensitivity problems when logging in.